WordPress/MediaEditor/MediaEditorHub.swift

```diff
@@ -0,0 +1,10 @@
+import UIKit
+
+class MediaEditorHub: UIViewController {
```

Comment:
Hello hub 👋 Who are you? Why are you here? :P

Reply:
The Hub doesn't make much sense in this PR, but he'll be responsible for:

* Showing a loading indicator if we're waiting for an image to be available
* Show the multiple media editor capabilities buttons (Basics, Brightness, Filter)
* Show multiple images so the user can select which one he wants to manipulate

You can see that on the first screen on PR's description!